airflow/contrib/hooks/gcp_dataflow_hook.py

```diff
@@ -369,7 +379,7 @@ def start_template_dataflow(self, job_name, variables, parameters, dataflow_temp
             name, variables, parameters, dataflow_template)
 
     def start_python_dataflow(self, job_name, variables, dataflow, py_options,
-                              append_job_name=True):
+                              py_interpreter=None, append_job_name=True):
```

Comment:
I don't think we need to explicitly check for py_interpreter for a number of reasons:
a. it should be left to the users which version python points to. 
b. airflow is about to deprecate py2 support soon.

Reply:
while airflow may be ready to deprecate python2 support, most python beam pipelines are authored in python2.

Are you suggesting `py_interpreter` should be a required argument rather than defaulting to python2?

Reply:
Ok 30fe1b7 adds a change to infer default`py_interpreter` from [`sys.version_info`](https://docs.python.org/3/library/sys.html#sys.version_info).